src/osvr/Server/ConfigFilePaths.cpp

```diff
@@ -77,20 +77,26 @@ namespace osvr {
             configDir /= fs::path("OSVR") / configSubpath;
 #endif
 
-            return configDir.string();
+#if defined(OSVR_ANDROID)
+            configDir = fs::path("/sdcard/osvr");
+#endif
+            auto ret = configDir.string();
```

Comment:
Any particular reason you split this up? (Does `.string()` return a reference or something weird?)  It's not a blocker, just curious.